Assets/Scripts/2D/SliderControlsScript.cs

```diff
@@ -60,15 +60,15 @@ void Update()
         {
             if (!InputField.isFocused)
             {
-                Manager.InputFieldInUse = false;
+                Manager.DisableShortcuts = false;
```

Comment:
Appropriate name change

Reply:
Thanks. I felt 'InputFieldInUse' was too specific for the purpose of the flag